src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -23,6 +23,8 @@ const QString kSingleEq = "SingleEQEffect";
 const QString kDefaultEqId = BiquadFullKillEQEffect::getId();
 const QString kDefaultMasterEqId = QString();
 const QString kDefaultQuickEffectId = FilterEffect::getId();
+//: Shown when no EQ is loaded
+const QString kNoEffectLoadedLabel = QObject::tr("None");
```

Comment:
recently I used  [Disambiguation string](https://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation) to add a hint for translators. Then I discovered `//:` is similiar.
What's the difference for transifex translators?